components-api/src/main/scala/pl/touk/nussknacker/engine/api/json/encoders/ToJsonEncoderWithFallback.scala

```diff
@@ -0,0 +1,108 @@
+package pl.touk.nussknacker.engine.api.json.encoders
+
+import cats.data.ValidatedNel
+import cats.implicits.{catsSyntaxValidatedId, toTraverseOps}
+import io.circe.{Encoder, Json}
+import io.circe.Encoder.encodeZonedDateTimeWithFormatter
+import io.circe.Json.{
+  fromBigDecimal,
+  fromBigInt,
+  fromBoolean,
+  fromDouble,
+  fromFloat,
+  fromInt,
+  fromLong,
+  fromString,
+  Null
+}
+
+import java.math.BigInteger
+import java.time._
+import java.time.format.DateTimeFormatter
+import java.util.UUID
+import scala.annotation.tailrec
+import scala.jdk.CollectionConverters._
+
+object ToJsonEncoderWithFallback {
```

Comment:
Added encoding for types, that were present in the ToJsonEncoder, but not here, and added ability to use fallback.